src/config/db.cofig.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+require('dotenv').config();
+
+const DB = process.env.MONGODB_URI;
```

Comment:
use like this // Load environment variables
const MONGO_USERNAME = process.env.MONGO_USERNAME;
const MONGO_PASSWORD = process.env.MONGO_PASSWORD;
const MONGO_DATABASE_NAME = process.env.MONGO_DATABASE_NAME;
const MONGO_CLUSTER_NAME = process.env.MONGO_CLUSTER_NAME;

// Construct the MongoDB URL
const MONGO_URL = `mongodb+srv://${MONGO_USERNAME}:${MONGO_PASSWORD}@${MONGO_CLUSTER_NAME}.mongodb.net/${MONGO_DATABASE_NAME}`;